global_body_planner/include/global_body_planner/functions.h

```diff
@@ -0,0 +1,122 @@
+#ifndef FUNCTIONS_H
+#define FUNCTIONS_H
```

Comment:
Is there a way to namespace these functions and #define's? We should also think about renaming it to something more descriptive?.